rust/osm2lanes-web/src/map.rs

```diff
@@ -9,10 +9,12 @@ use web_sys::{Element, HtmlElement, Node};
 use yew::prelude::*;
 use yew::Html;
 
+use crate::Msg as WebMsg;
+
 #[allow(clippy::large_enum_variant)]
 pub enum Msg {
     MapClick(LatLng),
-    MapUpdate(Tags, Locale, Vec<LatLon>),
+    MapUpdate(String, Tags, Locale, Vec<LatLon>),
```

Comment:
I'd vote for switching to ```MapUpdate { way_id: String, tags: Tags, locale: Locale, points: Vec<LatLon> }``` when either the tuple list grows or one of the types isn't obvious

Reply:
agreed, I did one but not the other :/